app/app.go

```diff
@@ -855,7 +856,14 @@ func (app *Evmos) LoadHeight(height int64) error {
 // ModuleAccountAddrs returns all the app's module account addresses.
 func (app *Evmos) ModuleAccountAddrs() map[string]bool {
 	modAccAddrs := make(map[string]bool)
-	for acc := range maccPerms {
+
+	accs := make([]string, 0, len(maccPerms))
```

Comment:
nit, why is sorting necessary?

Reply:
iteration over maps is non-deterministic and could lead to the chain halting. Here it's ok because we're not executing anything on-state but using another map